components_beta/template/my_component/frontend/src/streamlit/streamlit.ts

```diff
@@ -15,6 +15,7 @@
  * limitations under the License.
  */
 
+import { EventTarget } from "event-target-shim"
```

Comment:
Can you add a comment here explaining why this is needed? Like:

"Safari doesn't support the `EventTarget` class, so we use a shim"

Reply:
Will do!